tinygrad/nn/__init__.py

```diff
@@ -291,11 +291,16 @@ class RMSNorm:
   print(norm(t).numpy())
   ```
   """
-  def __init__(self, dim, eps=1e-6): self.eps, self.weight = eps, Tensor.ones(dim)
-
-  def _norm(self, x:Tensor): return x * (x.square().mean(-1, keepdim=True) + self.eps).rsqrt()
```

Comment:
Was this function overridden anywhere? Why change this?

Reply:
_norm specifically? The _norm method was only used within __call__ in both tinygrad and the referenced llama implementation. Integrating the normalization code directly into __call__ stylistically aligns better with the other nn layers.